### ISSUE TITLE
[Settings]: In scan mode, Screen Reader users are not able to open 'Open JSON file' control.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#12003]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win+Ctrl+ Enter' key. Turn ON Narrator scan mode using 'Caps lock + Space' key.
4. Now navigate to 'Open JSON file' control and try to activate it then observe the issue.

**User Experience:**
Screen Reader users will be impacted here as they are not able to activate a control in scan mode which will not enhance the UX of those users.

**Guideline Reference:**
EN 301 549 V3.2.1: 11.5.2.12 Execution of available actions

**Attachments:**
[Uploading In scan mode, Screen Reader users are not able to open 'Open JSON file' control..zip…]()

### Expected Behavior

In scan mode, Screen Reader users should be able to open 'Open JSON file' control using 'Enter' key.

### Actual Behavior

In scan mode, Screen Reader users are not able to open 'Open JSON file' control using 'Enter' key. Screen Reader Users is only able to open JSON file in Scan off mode.

**Comment by carlos-zamora** [comment]
Copying over some relevant information from #12319:
> > PowerToys v0.55 settings has a NavigationView. It doesn't have this issue.
XAML Controls Gallery also doesn't have this issue.

> Hmm. That might actually be on us then. PT Settings is also XAML Islands IIRC, so if their NavView works, then it's probably how we're using it.

There's a chance this is on https://github.com/microsoft/microsoft-ui-xaml, but we need to verify that first.
